Monika After Story/game/script-topics.rpy

```diff
@@ -3268,14 +3268,9 @@ label monika_japanese:
                 "No.":
                     m 1e "Oh I see. That's alright!"
                     m 4b "If you want to learn Japanese, here's a phrase I can teach you."
-                    if persistent.gender == "F":
-                        m 4k "{i}Aishiteru yo, [player]-chan{/i}."
-                        m 1j "Ehehe~"
-                        m 1e "That means I love you, [player]-chan."
-                    else:
-                        m 4k "{i}Aishiteru yo, [player]-kun{/i}."
-                        m 1j "Ehehe~"
-                        m 1e "That means I love you, [player]-kun."
```

Comment:
Your commit is deleting existing code. Ensure your branch is updated before applying your commits.

Reply:
got it. I did it